symfony/plugins/orangehrmTimePlugin/Api/Model/TimesheetActionLogModel.php

```diff
@@ -19,43 +19,55 @@
 
 namespace OrangeHRM\Time\Api\Model;
 
+use DoctrineProxies\__CG__\OrangeHRM\Entity\EmployeeTerminationRecord;
```

Comment:
what is the usage of this import?

Reply:
This should be changed to the `use OrangeHRM\Entity\EmployeeTerminationRecord;`